modules/ossm-threescale-applying-external-service-entry-objects.adoc

```diff
@@ -49,25 +96,52 @@ spec:
   location: MESH_EXTERNAL
   resolution: DNS
 ----
+.. Add the `DestinationRule` CR to a file called _destination-rule-threescale-saas-system.yml:_
 +
-You can use the `oc apply` command with either of the following methods to apply the objects:
+.DestinationRule CR
+[source,terminal]
+----
+apiVersion: networking.istio.io/v1beta1
+kind: DestinationRule
+metadata:
+  name: destination-rule-threescale-saas-system
+spec:
+  host: multitenant.3scale.net
+  trafficPolicy:
+    tls:
+      mode: SIMPLE
+      sni: multitenant.3scale.net
+----
+
+. Apply and save the external `ServiceEntry` CR for the 3scale Hosted system to your cluster, by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <service-entry-threescale-saas-system.yml>
```

Comment:
The `service-entry-threescale-saas-backend.yml` reference in this command is passed literally to the command line and it is not a variable. This is because the preceding substep requested that the file is saved explicitly using this file name. You could remove the parenthesis:

> $ oc apply -f service-entry-threescale-saas-system.yml